crates/egui/src/widgets/drag_value.rs

```diff
@@ -466,7 +466,7 @@ impl<'a> Widget for DragValue<'a> {
                 .memory_mut(|mem| mem.drag_value.edit_string.take())
                 .unwrap_or_else(|| value_text.clone());
             let response = ui.add(
-                TextEdit::singleline(&mut value_text)
+                TextEdit::singleline_button(&mut value_text, ui.layout().horizontal_align())
```

Comment:
Better to add `align` and `use_content_width` helpers instead of `singleline_button`

```suggestion
                TextEdit::singleline(&mut value_text)
                    .align(ui.layout().horizontal_align())
                    .use_content_width(true)
```